src/polymer/iron-flex-layout-classes.ts

```diff
@@ -0,0 +1,247 @@
+      let currentModules: string[] = [];
+      if (styleNode && dom5.hasAttribute(styleNode, 'include')) {
+        currentModules = dom5.getAttribute(styleNode, 'include')!.split(' ');
+      }
+      const missingModules = usedModules
+        .filter((m) => currentModules.indexOf(m.name) === -1)
+        .map((m) => m.name)
+        .join(' ');
+      if (!missingModules.length) {
+        continue;
+      }
+      const multi = missingModules.indexOf(' ') > -1;
+      const warning = new FixableWarning({
+        code: 'iron-flex-layout-classes',
+        message:
+        `${multi ? 'These' : 'This'} style module${multi ? 's are' : ' is'} used but not imported:
+
+  ${missingModules}
+
+Import ${multi ? 'them' : 'it'} in the template style include.`,
+        parsedDocument,
+        severity: Severity.WARNING,
+        sourceRange: parsedDocument.sourceRangeForStartTag(domModule)!
+      });
+      if (!styleNode) {
+        const indent = getIndentationInside(templateContent);
+        warning.fix = [{
+          replacementText: `\n${indent}<style include="${missingModules}"></style>`,
+          range: sourceRangeForPrependContent(parsedDocument, template)
+        }];
+      } else if (currentModules.length) {
+        warning.fix = [{
+          replacementText: ` ${missingModules}`,
+          range: sourceRangeForAppendAttributeValue(parsedDocument, styleNode, 'include')!
+        }];
+      } else {
+        warning.fix = [{
+          replacementText: ` include="${missingModules}"`,
+          range: sourceRangeForAddAttribute(parsedDocument, styleNode)
+        }];
+      }
+      warnings.push(warning);
+    }
+  }
+}
+
+function sourceRangeForAppendAttributeValue(parsedDocument: ParsedHtmlDocument, node: dom5.Node, attr: string) {
+  const tagRange = parsedDocument.sourceRangeForAttributeValue(node, attr)!;
+  return {
+    file: tagRange.file,
+    start: { line: tagRange.end.line, column: tagRange.end.column - 1 },
+    end: { line: tagRange.end.line, column: tagRange.end.column - 1 }
+  };
+}
+
+function sourceRangeForAddAttribute(parsedDocument: ParsedHtmlDocument, node: dom5.Node) {
+  const tagRange = parsedDocument.sourceRangeForStartTag(node)!;
+  return {
+    file: tagRange.file,
+    start: { line: tagRange.end.line, column: tagRange.end.column - 1 },
+    end: { line: tagRange.end.line, column: tagRange.end.column - 1 }
+  };
+}
+
+function sourceRangeForPrependContent(parsedDocument: ParsedHtmlDocument, node: dom5.Node) {
+  const tagRange = parsedDocument.sourceRangeForStartTag(node)!;
+  return {
+    file: tagRange.file,
+    start: { line: tagRange.end.line, column: tagRange.end.column },
+    end: { line: tagRange.end.line, column: tagRange.end.column }
+  };
+}
+
+registry.register(new IronFlexLayoutClasses());
+
+// TODO(valdrin) move this in commons or something.
+/* ---- START copy-pasted from content-to-slot-usages. --- */
+function elementSelectorToPredicate(simpleSelector: string): dom5.Predicate {
```

Comment:
This has merged in, so after rebase you should be able to move this into html/util